pkg/projects/dir.props

```diff
@@ -96,6 +96,38 @@
     </BuildRID>
   </ItemGroup>
 
+  <!-- ================== Section for Universal Windows Platform package ================== -->
```

Comment:
If I move it, then it would require a major refactoring because there are `ITEMGROUPS` bellow this that depend on what I'm setting, and there are `ITEMGROUPS` above this that are required in order to define my itemgroup